client/me/purchases/paths.js

```diff
@@ -6,30 +6,69 @@ export const addCreditCard = purchasesRoot + '/add-credit-card';
 
 export const billingHistory = purchasesRoot + '/billing';
 
-export function billingHistoryReceipt( receiptId = ':receiptId' ) {
+export function billingHistoryReceipt( receiptId ) {
+	if ( process.env.NODE_ENV !== 'production' ) {
+		if ( 'undefined' === typeof receiptId ) {
+			throw new Error( 'purchaseId must be provided' );
+		}
+	}
 	return billingHistory + `/${ receiptId }`;
 }
 
-export function managePurchase( siteName = ':site', purchaseId = ':purchaseId' ) {
+export function managePurchase( siteName, purchaseId ) {
+	if ( process.env.NODE_ENV !== 'production' ) {
```

Comment:
This check bit looks a bit repetitive. Do you think we can add a helper function here to reduce the code repetition?

Reply:
Definitely repetitive, but this mess encourages me to rip it back out soon, which is the plan 🙂